### ISSUE TITLE
Add optional on/off switch for sensors

### ISSUE THREAD

**Issue by SphtKr** [SphtKr/homebridge-zway#64]
Particularly motion sensors and contact sensors, which generate notifications. HomeKit doesn't let you turn the notifications on/off based on a scene/schedule, so we need a switch we can turn on/off to prevent them from alerting. Not perfect, but better than nothing.

**Comment by florian-muc** [comment]
I think this is problematic, as it will also cause triggers based on those sensor values to fail, not just (user) notifications to mute.

**Comment by SphtKr** [comment]
Yes, very true. It would come with caveats to that effect, at least.
